### PR TITLE
Feature/nucleus

### PR DESCRIPTION
**Pull request by pkkummermo**

### CODE REVIEW

File: config.nucleus.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    "components/**/*.scss"
+  ],
+  "target": "dist",
+  "css": "styles/styleguide.css",
+  "namespace": "page",
+  "verbose": 3,
+  "counterCSS": null,
+  "staticLipsum": false,
+  "placeholderService": false
+}
```
Comment (tipsy) [inline config.nucleus.json]: Newline

File: gulpfile.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+});
+
+gulp.task('sass:watch', () => {
+  gulp.watch(
+    [`${ componentsFolder }/*.scss`, `${ componentsFolder }/**/*.scss`],
+    ['sass', 'styleguide', 'connect:reload']);
+});
+
+gulp.task('styleguide:watch', ['styleguide', 'sass', 'connect', 'sass:watch'])
+gulp.task('default', ['styleguide', 'sass']);
```
Comment (tipsy) [inline gulpfile.js]: Newline

File: gulpfile.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
+'use strict';
```
Comment (tipsy) [inline gulpfile.js]: Format

File: config.nucleus.json (hunk 1 above)
Comment (pkkummermo) [inline config.nucleus.json]: Done!

File: gulpfile.js (hunk 3 above)
Comment (pkkummermo) [inline gulpfile.js]: Done!

**Pull request merged**